Utilities/Maintenance/UpdateRemoteModules.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+program_name=$0
+
+function usage {
+  echo "Usage: ${program_name} "
+  echo "  param1 Remote modules' path"
+  exit 1
+}
+
+# Display usage
+if [ ${#@} != 1 ]; then
+  usage
+fi
+
+remote_modules_path=$1
+
+git_repository_tag_label='GIT_REPOSITORY'
+git_tag_label='GIT_TAG'
+
```

Comment:
We could check here whether `jq` is available, and if it is not, provide a suggestion for how to install it!

Reply:
Done in b68921c.